chain/params.go

```diff
@@ -16,6 +16,7 @@ type Params struct {
 
 	// AllowList configuration
 	ContractDeployerAllowList *AllowListConfig `json:"contractDeployerAllowListConfig,omitempty"`
+	TransactionsAllowList     *AllowListConfig `json:"transactionsAllowListConfig,omitempty"`
```

Comment:
Minor nitpick, I would probably consider shortening the JSON element names to just `transactionsAllowList` and `contractDeployerAllowList`.

Reply:
Done in https://github.com/0xPolygon/polygon-edge/pull/1349/commits/a9ee37f3e386d6b41b8bb11e956076c54e0c725a